GameEngineTests/GameStateTests.cs

```diff
@@ -18,11 +18,11 @@ public class GameStateTests
         [TestInitialize]
         public void TestInitialize()
         {
-            Player = new LeastRecentCardPlayer();
             Multiplier = 6;
-            NumberOfOwls = 6;
-            Deck = new Deck(Multiplier);
-            State = new GameState(Player, Deck, Multiplier);
+            Player = new LeastRecentCardPlayer();
+            State = new GameState(Multiplier, Player);
+            Deck = State.Deck;
```

Comment:
💡 We're a little inconsistent accessing `Deck` vs `State.Deck`. Give this change (of Deck being a part of State), I prefer removing the `Deck` var and using `State.Deck` everywhere. Thoughts?

Reply:
Done.